hibernate-core/src/main/java/org/hibernate/persister/entity/Joinable.java

```diff
@@ -85,16 +85,16 @@ public default String filterFragment(String alias, Map<String, Filter> enabledFi
 		return filterFragment( alias, enabledFilters, Collections.emptySet() );
 	}
 
-	public default String filterFragment(TableGroup tableGroup, Map<String, Filter> enabledFilters) throws MappingException {
-		return filterFragment( tableGroup, enabledFilters, Collections.emptySet() );
-	}
-
 	/**
 	 * Get the where clause filter, given a query alias and considering enabled session filters
 	 */
 	public String filterFragment(String alias, Map<String, Filter> enabledFilters, Set<String> treatAsDeclarations) throws MappingException;
 
-	public String filterFragment(TableGroup tableGroup, Map<String, Filter> enabledFilters, Set<String> treatAsDeclarations) throws MappingException;
+	public String filterFragment(
+			TableGroup tableGroup,
+			Map<String, Filter> enabledFilters,
+			Set<String> treatAsDeclarations,
+			boolean useIdentificationVariable) throws MappingException;
```

Comment:
Because for table references in DML statements, we currently do not generate aliases, so we need a way to control whether the alias or the table name should be used.